cfme/tests/configure/test_workers.py

```diff
@@ -27,3 +54,42 @@ def test_restart_workers(appliance):
     wait_for(lambda: len(pids) == len(worker.get_all_worker_pids()),
              fail_func=worker.parent.reload_workers_page, num_sec=1800, delay=10,
              message="Wait for all original workers are back online")
+
+
+@pytest.mark.tier(0)
+@pytest.mark.parametrize("dropdown", DROPDOWNS, ids=IDS)
+@pytest.mark.meta(blockers=[BZ(1656873)])
+def test_set_memory_threshold_in_ui(appliance, dropdown):
+    """
+    Bugzillas:
+        * 1656873
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: WebUI
+        caseimportance: medium
+        initialEstimate: 1/30h
+        testSteps:
+            1. Navigate to Configuration
+            2. Select the workers tab
+            3. Set memory threshold of any dropdown to a value above 1 GB
+            4. Hit save
+        expectedResults:
+            1.
+            2.
+            3.
+            4. the change should be reflected in the dropdown
+    """
+    view = navigate_to(appliance.server, 'Workers')
+    mem_threshold = getattr(view.workers, dropdown)
+    before = mem_threshold.selected_option
+    mem_threshold.select_by_visible_text("1.1 GB")
+    view.workers.save.click()
+    view.wait_displayed()
+    after = mem_threshold.selected_option
+
+    assert before != after
```

Comment:
Not sure about this check here - defaults might get updated to 1.1GB per worker and then this fails.

Thought: check that `1.1` isn't what is already selected, select 1.2 instead if so

This way the assertion that after matches the value we selected is a complete assertion that it _changed_.

Reply:
While unlikely that this would be the default, I agree that it is good to check and make sure that this is not the case before executing the test.